samples/lcd/lcd-evt-logo/main.cpp

```diff
@@ -23,82 +23,73 @@ constexpr uint8_t deviceCount = 1;
 IO::GPIO* devices[deviceCount];
 
 unsigned char evtBitMap[1024] = {
```

Comment:
That makes sense, but like you said, it can't really be seen. Also if anything needs to be changed, it is going to be easier to just regenerate it instead of editing by hand.